src/appshell/qml/AppWindow.qml

```diff
@@ -54,13 +54,13 @@ ApplicationWindow {
         id: titleProvider
     }
 
-    MainWindowProvider {
-        id: windowProvider
+    MainWindowHelper {
```

Comment:
I propose to rename to `MainWindowQmlBridge` or just `MainWindowBridge`

Reply:
Done! I chose `MainWindowBridge`, because it's shorter and because it's a bidirectional bridge :)